parser/config.go

```diff
@@ -30,6 +31,8 @@ Use 'deis help [command]' to learn more.
 		return configPull(argv)
 	case "config:push":
 		return configPush(argv)
+	case "config:routing":
```

Comment:
Do we want to support two different methods of toggling routing? I think it might make sense not to have this at all to avoid confusion.